Modules/Segmentation/SignedDistanceFunction/wrapping/itkPCAShapeSignedDistanceFunction.wrap

```diff
@@ -1,7 +1,12 @@
 itk_wrap_class("itk::PCAShapeSignedDistanceFunction" POINTER)
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
+    # Force wrapping for double images as it is the default image template type in
+    # C++.
+    UNIQUE(types "${WRAP_ITK_REAL};D")
     # Can only be wrapped for `double` because of OptimizerParameters that
```

Comment:
This comment should now be modified.

Reply:
I guess it could be interpreted ambiguously as it adds `double` on top of other real types. I'll try to think of a better phrasing.

Reply:
@dzenanz : Let me know if the new phrasing is more explicit.